src/CheckBox/README.md

```diff
@@ -0,0 +1,17 @@
+# Checkbox Component
+
+Checkbox based off of the [WAI-ARIA authoring guidelines for the checkbox component](https://www.w3.org/TR/wai-aria-1.1/#checkbox). The checkbox is an HTMl input element with added attributes to ensure proper functionality and accessibility. The checkbox is wrapped in an `asInput` component. The `asInput` wrapper is passed the input element as well as a second parameter set to `false` to ensure that the label is placed to the right of the checkbox.
+
+The following inputs should be passed into every checkbox component:
+* `name`: `name` attribute - type `String`
+* `label`: label to be placed next to the checkbox - type `String`
+* `checked`: `"true"` if the default state should be checked, `"false"` otherwise - type `String`
```

Comment:
I'm don't think there is an actual reason. I changed it to a boolean which makes more sense and also allows me to write more concise code.